packages/terra-popup/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ Unreleased
 ### Changed
 * Prevent click events from propagating from popup overlay
 
+### Fixed
+* Removed 'close' class from _PopupContent button
```

Comment:
I believe you might want to wrap _PopupContent in `` (i.e. `PopupContent`).  Otherwise, I think the markdown will treat this as italicized